ebusd-2.1.x/de/vaillant/08.ehp.csv

```diff
@@ -174,7 +174,7 @@ r,,HwcModeFulfilledHours,DHW_Mode_Fulfilled_Hours,,,,"A700",,,hoursum,,,total ti
 r,,CompState,CompressorState,,,,"E400",,,UCH,,,compressor and ICL state machine status number
 r,,TimeCompOnMin,MinTimeCompressorOn,,,,"E600",,,seconds,,,"minimum time that compressor must run in 0,5s steps"
 r,,TimeCompOffMin,MinTimeCompressorOff,,,,"E700",,,seconds,,,"minimum time the compressor must be off before restart in 0,5s steps"
-r,,TimeBetweenTwoCompStartsMin,Min. Pausenzeit Kompressor,,,,"E800",,,seconds,,,"calculated from menu item ""compressor starts per hour"" in 0,5s steps"
+r,,TimeBetweenTwoCompStartsMin,Min. Pausenzeit Kompressor,,,,"E800",,,seconds,,,"calculated from menu item """"compressor starts per hour"""" in 0,5s steps"
```

Comment:
double double escaping just because some client can't parse it?
I can't accept this

Reply:
OK - it was merely a quick and easy suggestion to solve [issue #351](https://github.com/john30/ebusd/issues/351).
I notice that you've added an enhancement tag to my [other suggestion](https://github.com/john30/ebusd/issues/351) - thanks for that.

Reply:
solved with [02d2845](https://github.com/john30/ebusd/commit/02d284587881ebd547fd7d81fe485075b6b1c04f), so please remove these lines from de+en